custom/ewsghana/handler.py

```diff
@@ -8,6 +8,14 @@
 from custom.ilsgateway.tanzania.handlers.notsubmitted import NotSubmittedHandler
 
 
+def looks_like_prod_code(code):
```

Comment:
Basically it's copy pasted from: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/commtrack/sms.py#L319
I need to handle this kind of message in custom module in order to send appropriate messages.